src/server/editorServices.ts

```diff
@@ -5,6 +5,8 @@
 module ts.server {
     export interface Logger {
         close(): void;
+        isVerbose(): boolean;

```

Comment:
these will require changes on the VS side as well to implement the new APIs. alternativelly you can use one of the methods of the old Logger interface that VS still implements:
```ts
        bool information();
        bool debug();
        bool warning();
        bool error();
        bool fatal();
```